cli/flox-rust-sdk/src/providers/publish.rs

```diff
@@ -3,36 +3,28 @@ use std::str::FromStr;
 
 use catalog_api_v1::types::{Output, Outputs, SystemEnum};
 use chrono::{DateTime, Utc};
-use flox_core::canonical_path::CanonicalPath;
+use log::trace;
```

Comment:
(non-blocking, same for `providers::catalog`) Should these be `debug`? Trace is pretty noisy.